Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -1507,19 +1507,19 @@ MenuName="Falke"
 ; EN: DeathMessage="%o was telefragged by %k."
```

Comment:
More uneditable:

Line 1489: "Du bist tot. Drücke [Feuer], um wieder zu erscheinen!" → "Du bist tot. Drücke [Schießen], um erneut zu spawnen!" (Actually, which one is it now in options? Feuer or Schießen? Would suggest to stick with options-terminology)

Line 1491: "Sie sehen zu. Drücke [Feuer], um von einem anderen Spieler zu sehen!" → "Du schaust zu. Drücke [Schießen], um von einem anderen Spieler aus zuzusehen!"